UsageStatistics_Page.php

```diff
@@ -61,12 +61,12 @@ public function render() {
 			);
 
 			include  W3TC_DIR . '/UsageStatistics_Page_DbRequests_View.php';
-		} elseif ( isset( $_REQUEST['view'] ) && $_REQUEST['view'] == 'oc_requests' ) {
+		} elseif ( isset( Util_Request::get_string( 'view' ) ) && Util_Request::get_string( 'view' ) == 'oc_requests' ) {
 			$storage = new UsageStatistics_StorageReader();
 			$summary = $storage->get_history_summary();
 			$timestamp_start = $summary['period']['timestamp_start'];
 
-			$sort_column = isset( $_REQUEST['sort'] ) ? $_REQUEST['sort'] : '';
+			$sort_column = isset( Util_Request::get_string( 'sort' ) ) ? Util_Request::get_string( 'sort' ) : '';
```

Comment:
Use `! empty()`.  You may want to set a variable to avoid calling twice.

Reply:
Changed to ! empty and created a variable to avoid 2 function calls.